source/documentation/index.html.md.erb

```diff
@@ -11,48 +11,34 @@ which covers service design more broadly.
 
 ## Principles
 
-{% assign principle_groups = site.pages
```

Comment:
Is it possible to maintain the dynamic generation of this page?

Reply:
I'm not sure, TBH. tech-docs is based on middleman rather than jekyll, but I imagine middleman has the same kind of features for doing that. I'll look into it.